src/main/java/bio/terra/service/tabulardata/google/bigquery/BigQueryDatasetPdao.java

```diff
@@ -129,6 +131,50 @@ public void createTable(
     bigQuery.create(buildLiveView(bigQueryProject.getProjectId(), datasetName, table));
   }
 
+  public void createColumn(
+      String datasetName, BigQuery bigQuery, DatasetTable table, Column column) {
+    // Shamelessly pulled from the BigQuery documentation
+    Table bigQueryTable = bigQuery.getTable(TableId.of(datasetName, table.getRawTableName()));
+    Schema schema = bigQueryTable.getDefinition().getSchema();
+    FieldList fields = schema.getFields();
+
+    boolean fieldExists =
+        fields.stream().anyMatch(field -> field.getName().equalsIgnoreCase(column.getName()));
+    if (!fieldExists) {
+      // Create the new field/column
+      Field newField = Field.of(column.getName(), translateType(column.getType()));
+      // Create a new schema adding the current fields, plus the new one
+      List<Field> newFields = new ArrayList<>(fields);
+      newFields.add(newField);
+      Schema newSchema = Schema.of(newFields);
+
+      // Update the table with the new schema
+      Table updatedTable =
+          bigQueryTable.toBuilder().setDefinition(StandardTableDefinition.of(newSchema)).build();
+      updatedTable.update();
+    } else {
+      logger.warn(
+          "Column {} already exists in table {}", column.getName(), table.getRawTableName());
```

Comment:
That's a great point! We can remove columns by exporting and loading the data into a new table: https://cloud.google.com/bigquery/docs/manually-changing-schemas#api_2. We would also need to create a view with the new table and update the raw table name stored in postgres. 

I'm debating whether we should handle this here or in a follow-on PR...I'll take a look at how big of a change this would be 👀

Reply:
Nice!  Seeing as this PR creates new functionality but does not yet expose it, I'd be fine to save this for a follow-up PR if you prefer.  May make review easier too!  Though it would be nice to have that follow-up live by the time the functionality is exposed in the API.

Reply:
I made a comment about this above but basically, I think it's reasonable to fail in the case where the column hasn't been successfully created but the user is then trying to create it with a different type.  I think if the type is the same, it's fair to swallow the exception and keep on going with the flight